README.md

```diff
@@ -1,69 +1,130 @@
 # gitwize-fe
-GitWize - Engineering Intelligence Tool - FrontEnd
 
-## Available Scripts
+![gitwizestats](images/gitwize-fe.png)
```

Comment:
Maybe this image could be a little bit further down, in case it occupies too much screen real state on the first load and hides that we have more content further down?

Reply:
I was wondering the same, how it might look like? If it happens as you mentioned, I will remove it in the next PR. What do you think?